common/socks5-client-core/src/lib.rs

```diff
@@ -188,12 +183,52 @@ impl NymClient {
         res
     }
 
-    pub async fn start(self) -> Result<TaskManager, Socks5ClientCoreError> {
-        #[cfg(not(target_os = "android"))]
-        let base_builder = self.create_base_client_builder().await?;
+    // async fn create_base_client_builder2(
```

Comment:
note: we should probably remove this dead code 

(I guess that's totally my fault lol)